### ISSUE TITLE
f-string-without-interpolation false negative (concatenation)

### ISSUE THREAD

**Issue by belm0** [PyCQA/pylint#3857]
### Steps to reproduce
```python
x = 10
y = f'foo' f'{x}'
z = f'{x}' f'foo'
```

### Current behavior
no errors

### Expected behavior
2nd and 3rd lines should emit `f-string-without-interpolation`

### pylint --version output
```
pylint 2.6.0
astroid 2.4.2
Python 3.7.6 (default, Dec 30 2019, 19:38:36)
[Clang 10.0.0 (clang-1000.11.45.5)]
```

**Comment by hippo91** [comment]
@belm0 thanks for the report.
I can reproduce it.

**Comment by DanielNoord** [comment]
@Pierre-Sassoulas Similar to the other message: while looking for f-string issues I found this. Screenshot shows that I can't reproduce this locally with the latest version of pylint.

<img width="1044" alt="Schermafbeelding 2021-08-02 om 16 25 11" src="https://user-images.githubusercontent.com/13665637/127877056-ef6468b1-71cf-40ef-9cbb-3a1ac2894570.png">

**Comment by Pierre-Sassoulas** [comment]
Thank you for helping me clean up, this is the most thankless aspect of being a maintainer (especially with 700+ open issues).

**Comment by ckclark** [comment]
> @Pierre-Sassoulas Similar to the other message: while looking for f-string issues I found this. Screenshot shows that I can't reproduce this locally with the latest version of pylint.
> 
> <img alt="Schermafbeelding 2021-08-02 om 16 25 11" width="1044" src="https://user-images.githubusercontent.com/13665637/127877056-ef6468b1-71cf-40ef-9cbb-3a1ac2894570.png">

@DanielNoord You did reproduce it.  The issue is about false-negative of `f-string-without-interpolation` check, and it is still reproducible at main branch @[bfa98ac](https://github.com/PyCQA/pylint/commit/bfa98ac131e5b7a1d91e66bd0ab7d7b3723a0da0) and 2.15.5 release.

**Comment by clavedeluna** [comment]
I'm wondering why we would want to flag the test cases mentioned in the issue with `f-string-without-interpolation` given that when evaluated it does evaluate to a form of interpolation:

```
>>> x = 10
>>> y = f'foo' f'{x}'
>>> z = f'{x}' f'foo'
>>> y
'foo10'
>>> z
'10foo'
```

so even though both `y` and `z` have two parts, they're concatenated.

**Comment by DanielNoord** [comment]
Can `y` be `'foo' f'{x}'`? Or can't you auto add an f-string with a normal string?

**Comment by clavedeluna** [comment]
> Can `y` be `'foo' f'{x}'`? Or can't you auto add an f-string with a normal string?

I don't think I understand your question. Can you give examples?

**Comment by DanielNoord** [comment]
I tested this myself. It is indeed a false negative:
`f'foo' f'{x}'` should/could be `'foo' f'{x}'`

**Comment by clavedeluna** [comment]
Wondering what an approach to fixing this would be, given that

```
y = f'foo' f'{x}'
(Pdb) node.values
[<Const.str l.4 at 0x10c640460>, <FormattedValue l.4 at 0x10c6402b0>]
```

and 
```
good_y = 'foo' f'{x}'
(Pdb) node.values
[<Const.str l.5 at 0x10c640250>, <FormattedValue l.5 at 0x10c6404f0>]
```

so both `f'foo'` and `'foo'` register as Const.str nodes, so there's no obvious way to distinguish them. Is this something that needs to happen in astroid?

**Comment by DanielNoord** [comment]
Well that's interesting. This might be hard to fix then: there is no good way to determine what qualifiers a string has before it (`r`, `f` or any others).
I didn't know the `ast` automatically transformed them into `Const`.
